crates/nu-protocol/src/shell_error.rs

```diff
@@ -91,6 +91,15 @@ pub enum ShellError {
         span: Span,
     },
 
+    /// A command received an argument with correct type but incorrect value.
+    ///
+    /// ## Resolution
+    ///
+    /// Correct the argument value before passing it in or change the command.
+    #[error("Incorrect value.")]
+    #[diagnostic(code(nu::shell::incorrect_value), url(docsrs))]
+    IncorrectValue(String, #[label = "{0}"] Span),
```

Comment:
Might be worth investigating if we already have similar variants that are poorly documented.